client/components/deposits-overview/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Card, CardHeader } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import { useAllDepositsOverviews } from 'wcpay/data';
+
+interface OverviewProps {
+	overview: AccountOverview.Overview;
+	account: AccountOverview.Account;
+	isLoading: boolean;
+}
+
+/**
+ * Renders a deposits overview
+ *
+ * @param {AccountOverview.Overview} props Deposits overview
+ * @return {JSX.Element} Rendered element with deposits overview
+ */
+const DepositsOverviewDetails: React.FunctionComponent< OverviewProps > = (
+	props
+) => {
+	return (
+		<Card>
+			<CardHeader>
+				{ __( 'Deposits', 'woocommerce-payments' ) }
```

Comment:
This is a good candidate for a `strings.ts` file within the component dir.

Or we can create this file in upcoming issues if needed.

Reply:
Thanks for the suggestion. I've added a start to this in [b8aaf2a](https://github.com/Automattic/woocommerce-payments/pull/5759/commits/b8aaf2a112178bcb24f2c286cf2ecee222835591)

My thinking with the structure of this `strings.ts` file would be something like:

```
export default {
	heading: __( 'Deposits', 'woocommerce-payments' ),
	next_deposits: {
		heading: __( 'Next deposits', 'woocommerce-payments' ),
		description: __( 'The amount may change while payments are still accumulating', 'woocommerce-payments' ),
	},
	deposits_history: {
		heading: __( 'Deposits history', 'woocommerce-payments' ),
	},
};
```